tests/attested_tls/sgxlkl_enclave/client.c

```diff
@@ -0,0 +1,92 @@
+
+/***************************************************************************
+  This is the entry point of sgxlkl_enclave which is responsible for:
+
+  1) Establishing a trusted channel with oe_enclave via clientAgent;
+  2) Verify the pregenerated cert (type = 1)
+
+ **************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+#include <unistd.h>
+#include <mbedtls/ssl.h>
+#include <mbedtls/net_sockets.h>
+#include "tlscli.h"
+#include "clientAgent.h"
+#include "../common/settings.h"
```

Comment:
Get rid of the dependency on settings.h and message.h, if there is any.

Reply:
message.h I think we may keep it since the server_port is used in multiple files like, client.c, host.cpp. But for now, I can just copy the server_port and remove the message.h.